documentation/Bundling.md

```diff
@@ -0,0 +1,483 @@
+# Bundling Azure SDK libraries for a browser
+
+To use Azure SDK libraries on a website, you need to convert your code to work inside the browser. You do this using a tool called a **bundler**. This process takes JavaScript code written using [Node.js](https://nodejs.org/) conventions and converts it into a format that is understood by browsers.
+
+This document will walk you through the steps required to bundle Azure SDK libraries for your website.
+
+## Install prerequisites
+
+In order to install Azure SDK libraries, you will need to install Node.js and a bunlder of your choice onto your development machine.
+
+### Node.js
+
+First, download and install Node.js from the official website: https://nodejs.org/en/
+
+Once it is installed correctly, you will be able to use it with the `node` command on the command-line:
+
+```
+node --version
+```
+
+### NPM
+
+The [Node Package Manager](https://npmjs.com) (npm) is included when you install Node. You can access it from the command-line, similar to Node:
+
+```
+npm --version
+```
+
+## Setting up your project
+
+If you already have a project with a package.json file set up, skip to the next section. If not, first let's make a new directory for your project, and change into it.
+
+```
+mkdir example
+cd example
+```
+
+Now let's [set up a package.json file](https://docs.npmjs.com/creating-a-package-json-file) to configure npm:
+
+```
+npm --init
```

Comment:
I think, the command is `npm init` without dashes (--). Also, what do you think about suggesting `npm init -y` to skip the prompts?

Reply:
Simpler seems better, I agree!